src/browser/transforms.js

```diff
@@ -172,6 +172,9 @@ function addBodyTrace(item, options, callback) {
         method: (!stackFrame.func || stackFrame.func === '?') ? '[anonymous]' : stackFrame.func,
         colno: stackFrame.column
       };
+      if (frame.method && frame.method.indexOf('f._wrapped') != -1) {
```

Comment:
This seems fragile - it might break if:

- this library changes, so the minified frame is no longer called `f._wrapped`
- this library is included in a larger application build, and the minified name there is something else

How about if we change the `_wrapped` part to be more identifiable? e.g. `_rollbar_wrapped` -- and then we can just check that the method name ends with the exact string `._rollbar_wrapped`